src/core/physics/collisions.rs

```diff
@@ -301,7 +301,7 @@ impl<'a> CollisionWorld<'a> {
             if !self.entities.is_alive(entity) {
                 // Remove any collisions with the killed entity from the collision cache.
                 let mut collisions = collision_cache.collisions.borrow_mut();
-                let colliding_with = collisions.remove(&entity);
+                let colliding_with = collisions.swap_remove(&entity);
```

Comment:
What is the effect of this change?

Reply:
Ah I should've commented on that - remove was deprecated in index-map crate due to ambiguity. Remove already did a `swap_remove`, they deprecated and require calling `swap_remove` or `shift_remove`, to clarify if user intends to remove and preserve ordering, or swap with last element for performance while modifying ordering.

Behavior has not changed, fixes deprecation warning. See context:

https://github.com/indexmap-rs/indexmap/blob/184fe4bfcba20e21242c924220170b98be157d45/src/map.rs#L645-L653